EL-3902 - Column grouping fix focus issue

darrenmcc-microfocus opened the pull request:
#### Checklist
<!-- Use an 'x' to check those which apply. -->
* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue
<!-- Either a Jira URL or a description of the issue that this PR addresses. -->
https://portal.digitalsafe.net/browse/EL-3902

#### Description of Proposed Changes
<!-- Describe what was changed and how it addresses the original issue. -->
This branch fixes two issues:

- When scrolling the deselected items in the column picker the scroll area comes into focus (outline now removed).
- Sometimes the list items within deselected also have a focus state that we do not want to see we have our own custom styling to show this (outline also removed).

#### Documentation CI URL
<!-- Initiate a build at https://jenkins.swinfra.net/job/SEPG/view/Templates/job/New%20SEPG%20Build/build -->
<!-- Append the branch name to the following URL: -->
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-3902-column-picker-grouping-focus-fix

#### Downstream Build(s)
<!-- Push a branch with the same name in each downstream repository and create a build in Jenkins. -->
<!-- Add the Jenkins build link(s) below: -->
* [caf/ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/view/Developer/job/caf~ux-aspects-micro-focus~EL-3902-column-picker-grouping-focus-fix~CI/)

buildmachine-sou-jenkins2 commented:
The Documentation QA site for this branch has been built:
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-3902-column-picker-grouping-focus-fix

donna-marijne commented:
Add a description please.

donna-marijne commented:
Downstream build also missing.
![image](https://user-images.githubusercontent.com/26114101/82306974-a84e7380-99b7-11ea-91d0-4a3b2f642986.png)

darrenmcc-microfocus commented:
> Downstream build also missing.
> ![image](https://user-images.githubusercontent.com/26114101/82306974-a84e7380-99b7-11ea-91d0-4a3b2f642986.png)

@donna-marijne this it loading for me, not sure why you are having issues.

darrenmcc-microfocus commented:
This branch fixes two issues: 

1. When scrolling the deselected items in the column picker the scroll area comes into focus (outline now removed).
2. Sometimes the list items within deselected also have a focus state that we do not want to see - we have our own custom styling to show this (outline also removed).

donna-marijne commented:
Neither of those two elements are focusable. Why does this fix the problem?

donna-marijne commented:
I started the downstream build by accident and it looks like it's wedged now, so I'll just merge this and we'll see what happens on the development branch.
